### DISCUSSION TITLE
Add @mouseenter and @mouseleave to q-item

### DISCUSSION THREAD

**Discussion by DerMrBacon** [quasarframework/quasar#17142] [Ideas / Proposals] [answered]
I would like to make a notification menu and if the user hovers/enters the q-item it should be possible to mark this item as read at least this is my usecase. But in general there should be such mouse events on this component.

Solution:
Add these events to the q-item

Question:
Is this a quasar issue or a vue issue?

**Comment by rstoenescu** [comment, marked as answer]
We don't need to manually add those or any other native ones. Just use them. Any event will get attached to the main node of any component.
